github_actions/spec/dependabot/github_actions/update_checker_spec.rb

```diff
@@ -392,6 +392,67 @@
       end
     end
 
+    context "given a realworld repository", :vcr do
+      let(:dependency) do
+        Dependabot::Dependency.new(
+          name: dependency_name,
+          version: dependency_version,
+          requirements: [{
+            requirement: nil,
+            groups: [],
+            file: ".github/workflows/main.yml",
+            source: dependency_source
+          }],
+          package_manager: "github_actions"
+        )
+      end
+      let(:dependency_name) { "deivid-rodriguez/github-action-push-to-another-repository" }
```

Comment:
suggest we move this to the dependabot-fixtures org

Reply:
Set as draft until I move the repo, thanks for the review and the suggestion!

Reply:
It's been moved now!